locales/en/fields.json

```diff
@@ -65,7 +65,7 @@
     "short_label": "To",
     "items": {
       "prison_transfer": {
-        "label": "Prison"
+        "label": "Prison or Young Offender Institution"
```

Comment:
I wonder if we should we be consistent with casing as below we write all in lowercase, but here we use capitals to suggest an acronym. Looking on GOV.UK, the [description for Feltham](https://www.gov.uk/guidance/feltham-yoi) uses all lowercase and includes the acronym:

> Feltham is a young offender institution (YOI) in London for boys and young men aged 15 to 21.

I wonder if we can use this pattern for the radio label and then use the acronym on the text label as the acronym has now been defined on the page:

```
Prison or young offender institution (YOI)
    Name of prison or YOI
```

Reply:
The caps in the radio label vs the lowercase in the conditional are definitely a bit weirdy-beardy

Reply:
@teneightfive That looks like a good suggestion - GOV.UK is prior art, and those changes deal with the "weirdy beardy" case issues.